src/Illuminate/Queue/Queue.php

```diff
@@ -288,4 +320,18 @@ public function setContainer(Container $container)
     {
         $this->container = $container;
     }
+
+    /**
+     * Raise the job queued event.
+     *
+     * @param string|object $job
+     * @param mixed $jobId
```

Comment:
can we do better than mixed?

Reply:
From https://github.com/imTigger/laravel-job-status: 

![image](https://user-images.githubusercontent.com/26772554/82471354-98f12800-9a94-11ea-8af5-e8eb79150ea9.png)

Reply:
Yep.